api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -30,6 +32,12 @@
  * @see ContextPropagators
  */
 public final class GlobalOpenTelemetry {
+  // Visible for testing
+  static final Logger logger = Logger.getLogger(GlobalOpenTelemetry.class.getName());
+
+  private static final boolean suppressSdkCheck =
+      Boolean.getBoolean("otel.sdk.suppress-sdk-initialized-warning");
```

Comment:
Why do we refer to SDK in the API?

Reply:
Because our SDK is going to be the solution for 99.99% of all users. I think it's fine to refer to our SDK like this.